docs/modules/reference/pages/performance-data-collection/collectors/SnmpCollector.adoc

```diff
@@ -89,6 +89,30 @@ Ignored when using Newts as time-series storage.
 .Configuration overview for SNMP data collection
 image::performance-management/collectors/01_snmp-datacollection-configuration.png[]
 
+=== Support for opaque data types
+
+OpenNMS supports opaque data types including MIB import and data collection.
+Opauqe is ASN.1(Abstract Syntax Notation One) binary encoded data which is additionally wrapped in a hex string.
+The extracted from out of a hex string data can be of various types. 
+At the moment is the support for float data type implemented, but it can be easily extended if necessary. 
+Implemented float format have 120 as type field in ASN.1.
```

Comment:
```suggestion
Implemented float format have 120 as type field in ASN.1.
```
When I looked at the code, there was a note that said "only the support for type 120 is at the time implemented". If we only support that, this should say:

Only float format `120` as the type field in ASN.1 is supported.

Reply:
Comment changed